index.html

```diff
@@ -183,35 +183,31 @@ <h3 id="unacceptablebehaviour">
+  <li><strong>Appreciate and Accommodate Our Similarities and Differences</strong>We come from many cultures and backgrounds, ways of life, and standard of behavior. Cultural differences can encompass everything from official religious observances to personal habits to clothing. Be respectful of people with different practices, attitudes, and beliefs. To help us achieve and maintain these high standards, each individual participant is expected to share responsibility for our work environment by adhering to the following behavioral guidelines:
+</li>
 	
-<p>To help us achieve and maintain these high standards, each individual participant is expected to share
-	responsibility for our work environment by adhering to the following behavioral guidelines:</p>
 	
-	<ol>
-		<li><strong>Respect.</strong> We are a large community of people who are passionate about our work, sometimes holding strong opinions beliefs. We are committed to dealing with each other with courtesy, respect, and dignity at all times. Misunderstandings and disagreements do happen. When conflicts arise, we are expected to resolve them maintaining that courtesy, respect, and dignity, even when emotions are heightened.</li>
+	<ul>
+		<li><strong>Respect.</strong> We are a large community of people who are passionate about our work, sometimes holding strong opinions and beliefs. We are committed to dealing with each other with courtesy, respect, and dignity at all times. Misunderstandings and disagreements do happen. When conflicts arise, we are expected to resolve them maintaining that courtesy, respect, and dignity, even when emotions are heightened.</li>
 		
-  <li>Never <a>harass</a> or <a>bully</a> anyone verbally, physically or 
-   <a data-lt="sexual harassment">sexually</a>.</li>
-  <li>Never <a>discriminate</a> on the basis of personal characteristics or 
-    group membership.</li>
-  <li>Communicate constructively and avoid <a>demeaning</a> or <a>insulting</a>
-    behavior or language.</li>
-  <li>Seek, accept, and offer objective work criticism, and <a>acknowledge</a> 
```

Comment:
The sense of correct acknowledgement of work is being removed here and not replaced. I think it's important to keep it.

The part about seeking, accepting and offering objective work criticism is also removed and not replaced; I don't think it is so important to keep here because it is about _what_ we do whereas this document's focus is on the _manner_ with which we do things. However the lack of documented motivation for this change means it is not clear that there is consensus in the PWETF for making it.

Reply:
Change made to add acknowledging the contributions of your peers, that's a good suggestion for the kind of work down within the W3C compared to other communities. 👍